sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/trace/_exporter.py

```diff
@@ -433,6 +436,13 @@ def _convert_span_to_envelope(span: ReadableSpan) -> TelemetryItem:
         span.attributes,
         lambda key, val: not _is_standard_attribute(key)
     )
+
+    # Standard metrics special properties
+    # Only add the property if span was generated from instrumentation that supports metrics collection
+    if span.instrumentation_scope is not None and \
+        span.instrumentation_scope.name in _INSTRUMENTATION_SUPPORTING_METRICS_LIST:
+        data.properties["_MS.ProcessedByMetricExtractors"] = "True"
```

Comment:
Could you explain why we add this to ALL the telemetry from the metrics-supporting instrumentation and not just that which corresponds to the requests/duration and dependencies/duration

Reply:
`exceptions/count` and `traces/count` are not included in the scope of this PR. I have a PR out in defining how those will be populated [here](https://github.com/aep-health-and-standards/Telemetry-Collection-Spec/pull/136)

Reply:
got it.